DataFormats/Common/interface/PtrVector.h

```diff
@@ -65,7 +65,7 @@ namespace edm {
 
     Ptr<T> const operator[](difference_type n) const {  // Otherwise the
       return base_->fromItr(iter_ + n);                 // boost::range
-    }                                                   // doesn't have []
+    }  // doesn't have []
```

Comment:
In principle the new formatting is "worse" here, but really the entire multiline comment should be moved to a single line above the function definition. Better to be done after this PR.